tests/test_homepage.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestHomePage():
+
+    @pytest.mark.nondestructive
+    def test_home_page(self, base_url, selenium):
```

Comment:
Ah, thanks for pointing that out, @MonicaG. I now understand why I didn't see an explicit check for the visibility of the login button in your first iteration. I agree that at this point you can remove the line from the test that asserts that the user is not logged in, as that isn't part of the requirement and is duplication of the check that the login is visible.